apps/daimo-mobile/src/view/sheet/GlobalBottomSheetView.tsx

```diff
@@ -0,0 +1,180 @@
+import { now } from "@daimo/common";
+import BottomSheet, {
+  BottomSheetBackdrop,
+  BottomSheetView,
+} from "@gorhom/bottom-sheet";
+import { BottomSheetDefaultBackdropProps } from "@gorhom/bottom-sheet/lib/typescript/components/bottomSheetBackdrop/types";
```

Comment:
nice, also had this in https://github.com/daimo-eth/daimo/pull/942/files#diff-f7cc689638a3a13b6de53a059fd15fbb7667c498a12b0356fbdb01897611bca2 if you want to pick and merge instead